coala_quickstart/interaction/Logo.py

```diff
@@ -0,0 +1,74 @@
+import textwrap
+from coala_quickstart.Strings import COALA_BEAR_LOGO, WELCOME_MESSAGES
+
+
+def print_side_by_side(printer,
+                       left=[],
+                       right=[],
+                       left_color="white",
+                       right_color="blue",
+                       limit=80):
+    """
+    Prints the the given lines side by side. Example usage:
+
+    >>> from pyprint.ConsolePrinter import ConsolePrinter
+    >>> printer = ConsolePrinter()
+    >>> print_side_by_side(
+    ...     printer,
+    ...     ["Text content on the left",
+    ...      "side of the text."],
+    ...     ["Right side should contain",
+    ...      "this."],
+    ...     left_color=None,
+    ...     right_color=None,
+    ...     limit=80)
+    Text content on the left Right side should contain
+    side of the text.        this.
+
+    If either side is longer than the other, empty lines will
+    be added to the shorter side.
+
+    :param printer:
+        A ``ConsolePrinter`` object used for console interaction.
+    :param left:
+        The lines for the left portion of the text.
+    :param right:
+        The lines for the right portion of the text.
+    :param left_color:
+        The color to use for the left side of the text.
+    :parm right_color:
+        The color to use for the right side of the text.
+    :param limit:
+        The maximum line length limit.
+    """
+    max_length = limit - len(max(left, key=len))
+
+    for line in range(len(left) - len(right)):
+        right.append("")
+    for line in range(len(right) - len(left)):
+        left.append("")
+
+    for left_line, right_line in zip(left, right):
+        printer.print(left_line, color=left_color, end="")
+        printer.print(" " * (limit + 1 - max_length - len(left_line)), end="")
```

Comment:
`limit - max_length` is `limit - limit + len(...)`, considering that limit only exists for this expression I'd rather recommend putting the `len(...)` expression into a variable